build/jobs/add-aad-test-environment.yml

```diff
@@ -62,3 +62,5 @@ steps:
 
       Write-Host "Imported modules"
       $output = Add-AadTestAuthEnvironment -TestAuthEnvironmentPath $(System.DefaultWorkingDirectory)/testauthenvironment.json -EnvironmentName $environmentName -ResourceGroupName $(ResourceGroupName) -TenantAdminCredential $adminCredential -TenantId $tenantId
+      Write-Host "Adding Multi Tenant auth values"
+      $output2 = Add-AadTestAuthEnvironment -TestAuthEnvironmentPath $(System.DefaultWorkingDirectory)/testauthenvironment.json -EnvironmentName $environmentName -ResourceGroupName $(ResourceGroupName) -TenantAdminCredential $adminCredential -TenantId $tenantId -FhirServiceAppRegistrationOverride "https://$environmentName.azurewebsites.net/"
```

Comment:
Should we use the resourceTenantDomainSuffix variable instead of hardcoding azurewebsites.net?

Reply:
Yep, good point. Updated.